markdown/1.x/en/ff-checkout-tracking.md

```diff
@@ -4,10 +4,12 @@ The `ff-checkout-tracking` element sends out a checkout tracking request
 for each `ff-checkout-tracking-item` in its DOM. These elements are supposed
 to be generated on server side after the checkout is completed.
 Each product needs its own `ff-checkout-tracking-item` with its respective
-`record-id` and `count`.
+`record-id`. Optionally you can also provide a `count` and a `price` attribute.
 
+The `count` attribute specifies how many items of the product were bought. It defaults to 1.
+
+The `price` attribute specifies the price of the bought item. If this attribute is not present, it will be inferred from the clicked item and the configured [fieldRoles](/documentation/3.x/field-roles).
```

Comment:
```suggestion
The `price` attribute specifies the price of the bought item. If this attribute is not present, it will be inferred from the clicked item and the configured [fieldRoles](/documentation/1.x/field-roles).
```